src/pages/ReviewResults.js

```diff
@@ -0,0 +1,95 @@
+import {useEffect, useState} from 'react';
+import {div} from 'react-hyperscript-helpers';
+import {AccessReviewHeader} from './access_review/AccessReviewHeader';
+import {Election} from '../libs/ajax';
+import {getDarData, Notifications} from '../libs/utils';
+import {isNil} from 'lodash/fp';
+import {DarApplication} from "./access_review/DarApplication";
+
+const SECTION = {
+  margin: '20px',
+};
+
+export default function ReviewResults(props) {
+  const [match] = useState(props.match);
+  const [darId] = useState(props.match.params.referenceId);
```

Comment:
These two things don't have to be in state - they can be plain constants derived directly from the props.